ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -95,6 +95,7 @@ conformance_test(
             "DeeplyNestedValueIT",  # FIXME: Too deeply nested values flake with a time out (half of the time)
             "ExceptionsIT:ExCKRollbackGlobalArchivedCreate",  # TODO Temporarily disabled until Canton is bumped to a new Engine
             "ExceptionsIT:ExCKRollbackGlobalArchivedLookup",  # TODO Temporarily disabled until Canton is bumped to a new Engine
+            "ExceptionsIT:ExRollbackCreate",  # TODO Temporarily disabled until Canton is bumped to a new Engine
```

Comment:
Thanks for the ping! :)

Reply:
@rgugliel-da You are a code owner of the file, therefore github add you automatically as reviewer.